blocks/share-bar-block/features/share-bar/default.test.jsx

```diff
@@ -29,19 +32,30 @@ describe("When the share bar is shown", () => {
 				phrases={mockPhrases}
 			/>
 		);
-		expect(wrapper.find(".ts-share-bar__button")).toHaveLength(5);
+
+		const buttons = container.querySelectorAll(".b-share-bar button");
```

Comment:
We shouldn't be querying by selectors with RTL (React Testing Library) - we should use `getByRole` to query for specific buttons - see https://testing-library.com/docs/queries/about#priority - for details

Reply:
@matthewroach : Thanks! That was very helpful to know! There unfortunately wasn't anything visible to the user that identified the buttons except for the SVGs. I took out all the `querySelector` queries and replaced with the following:
1. All buttons selected: https://github.com/WPMedia/arc-themes-blocks/blob/TMEDIA-736-share-bar-block/blocks/share-bar-block/features/share-bar/default.test.jsx#L35-L36
2. Some buttons selected: https://github.com/WPMedia/arc-themes-blocks/blob/TMEDIA-736-share-bar-block/blocks/share-bar-block/features/share-bar/default.test.jsx#L57-L58